module_loadex_test.go

```diff
@@ -0,0 +1,52 @@
+package redis_test
+
+import (
+	. "github.com/bsm/ginkgo/v2"
+	. "github.com/bsm/gomega"
+
+	"github.com/redis/go-redis/v9"
+)
+
+var _ = Describe("Loadex Config", func() {
+	It("converts the configuration to a slice of arguments correctly", func() {
+		conf := &redis.ModuleLoadexConfig{
+			Path: "/path/to/your/module.so",
+			Conf: map[string]interface{}{
+				"param1": "value1",
+				"param2": "value2",
+				"param3": 3,
+			},
+			Args: []interface{}{
+				"arg1",
+				"arg2",
+				3,
+			},
+		}
+
+		args := conf.ToArgs()
+
+		// Test if the arguments are in the correct order
+		expectedArgs := []interface{}{
+			"MODULE",
+			"LOADEX",
+			"/path/to/your/module.so",
+			"CONFIG",
+			"param1",
```

Comment:
Check the order once @ktsivkov 
The actions are failing, you can modify it accordingly
I'll be happy to raise a PR for the same

Reply:
I guess my assumption was wrong about this
In Go, the iteration order over map keys is not guaranteed to be consistent, so the order of the configuration parameters in the output might not be the same as in the expected output. 
Ignore my previous comment

Maybe testing this wouldn't be the right approach
@ktsivkov

Reply:
@SoulPancake  yes, I saw that and removed the extra arguments on the config option, just one, so we can check it is transformed into a slice as expected.